podcastparser.py

```diff
@@ -744,6 +782,9 @@ def parse_pubdate(text):
     'rss/channel/item/enclosure': Enclosure('length'),
     'rss/channel/item/psc:chapters': PodloveChapters(),
     'rss/channel/item/psc:chapters/psc:chapter': PodloveChapter(),
+    'rss/channel/item/podcast:transcript': EpisodeAttrFromUrl("transcript"),
```

Comment:
As it's a URL to the transcript and not the transcript itself, `transcript_url` might be a better attr name?

Reply:
Done